### PR TITLE
Update magma contract

### PR DESCRIPTION
**Pull request by magmastone**
**NOTE**

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Please enable "Allow edits by maintainers" while putting up the PR.
3. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
4. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
5. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
6. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
7. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---

Update Magma contracts

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/magma exports TVL: 
        
 
 
    iotex                     426.09 k
    
    total                    426.09 k

File: projects/magma/index.js
Comment (g1nt0ki) [inline projects/magma/index.js:1]: can you change code to this, this way history is preserved/refill works

    const { getLiquityTvl } = require('../helper/liquity')
    // const { staking } = require('../helper/staking')
    const sdk = require('@defillama/sdk')
    
    module.exports = {
      iotex: {
        tvl: sdk.util.sumChainTvls([
          getLiquityTvl('0x21d81DABF6985587CE64C2E8EB12F69DF2178fe2'),
          getLiquityTvl('0xAeB0B38040aDdc4a2b520919f13944D9bC944435'),
        ]),
        // staking: staking('0x037a2e9a464fbA409D0E55600836864B410d6Dd8', '0x6C0bf4b53696b5434A0D21C7D13Aa3cbF754913E'), // disabled as backing of WEN is already counted towards tvl
      },
    };

**Review by magmastone — COMMENTED** [review]
that's good

**Comment by llamatester** [comment]
The adapter at projects/magma exports TVL: 
        
 
 
    iotex                     470.67 k
    
    total                    470.67 k

**Comment by llamatester** [comment]
Error while running adapter at projects/magma: 
        
 
  
    
    
    Error: Protocol doesn't have total tvl, make sure to export a tvl key either on the main object or in one of the chains
    Truncated error stack:
    
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:217:11

**Comment by llamatester** [comment]
Error while running adapter at projects/magma: 
        
 
  
    
    
    Error: Protocol doesn't have total tvl, make sure to export a tvl key either on the main object or in one of the chains
    Truncated error stack:
    
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:217:11

**Comment by llamatester** [comment]
The adapter at projects/magma exports TVL: 
        
 
 
    iotex                     479.18 k
    
    total                    479.18 k

**Pull request merged**